src/components/common/uploader/Uploader.tsx

```diff
@@ -34,7 +34,10 @@ function Uploader({ onChangeFile, isDisabled, fileExtensions, maxSize }: Uploade
   const handleClick = (e: MouseEvent<HTMLButtonElement>) => {
     e.preventDefault();
 
-    if (!isDisabled) inputRef.current?.click();
+    if (inputRef.current && !isDisabled) {
+      inputRef.current.value = '';
+      inputRef.current.click();
+    }
```

Comment:
```suggestion
if (!inputRef.current || isDisabled) return;
  
  inputRef.current.value = '';
  inputRef.current.click();
...
```

이 부분도 early return 처리될 수 있있을 거 같아요.

Reply:
[651e71a](https://github.com/JECT-Study/JECT-Official-WebSite-Client/pull/109/commits/651e71aef897a25b667835d8e09ec3e055c0a48c) 처리했습니다!